src/App.js

```diff
@@ -21,8 +21,19 @@ class App extends React.Component {
           recordCount={this.state.feed.recordCount}
           planRecordLimit={this.state.feed.tier}
         />
-        <BarChart data={this.state.feed.data} />
-        <NoticeBox notices={this.state.feed.notices}></NoticeBox>
+        <BarChart
+          data={this.state.feed.data}
```

Comment:
I see mentions of `feed` a lot - is this left over from a previous example? Should we use something more consistent with the data like `records` or `recordInfo` perhaps (to match your existing `planInfo`)?

Reply:
Yep, this is just what I named it at the beginning as it made sense to me at the time. Was on the todo list to update, I think recordInfo makes good sense 👍